release-notes.txt

```diff
@@ -1,3 +1,521 @@
+
+Networking
+----------
+    + sys: Add (CoAP) Link Format module
+    + sys: add credman (D)TLS credential management module
+    + sys: new sock submodule for DTLS
+    + gnrc_ipv6_ext_frag: Initial import of IPv6 fragmentation and reassembly
+    + gnrc_netif: allow for wait of minimum time between sends
+    * gnrc_netif: add netif setter (part of refactoring to a pointer-based
+      approach)
+    * gnrc_sixlowpan_frag_vrb: add gnrc_sixlowpan_frag_vrb_from_route()
+    + sixlowpan_sfr: initial import Selective Fragment Recovery definitions
+    + gcoap: add Blockwise support
+    * nanocoap: complete Blockwise support
+    + nanocoap: add coap_opt_get_next() to iterate over options
+    + nanocoap: add coap_opt_get_opaque() to retrieve option bytes
+    * tcp: several small improvements
+
+Packages
+--------
+    * ccn-lite: version bump
+    * pkg/libcose: bump version and update crypto support
+    * nimble: adapt to nrf51 family
+    + nimble: add IP-over-BLE support via netif/GNRC
+    * nimble: bump version to 1.2.0
+    * nimble/netif: set max conn explicitly
+    * semtech-loramac: add uplink_counter get/set functions
+    + Cifra: add AES crypto library
+    * cayenne-lpp: bump version to 0.1.1
+    + wolfSSL pkg addition with examples
+
+Boards
+------
+    + board: add support for SAMR34-XPRO
+    + boards/arduino-mkrwan1300: add initial support (without LoRa)
+    * boards: Split off 128KiB version of bl*pill
+    + boards/doc: Add a guide to find physical board pins
+    + boards: Initial support for board HiFive1B
+    - boards/mips-malta: remove board.
+    + boards/nrf52: introduce shared dependencies in common/nrf52 and update
+      boards
+    * boards/nrf52832-mdk: enable I2C and use default configuration
+    * board/nucleo-l031k6: Add i2c to periph_conf
+    * board/nucleo-l432kc: Implementing I2C for board
+    * boards: move CPU/CPU_MODEL definition to Makefile.features
+    + boards/p-l496g-cell02: add support
+    - boards: Remove support for the Jiminy-Mega256RFR2
+    * boards/sam0: refactor ADC_0 into plain ADC
+    * boards/slwstk6000b-*: split one board for each module
+    * boards/slwstk6000b: move CPU definiton to Makefile.features
+    + boards/stm32f723e-disco: add initial support
+    * boards/stm32: introduce common timer configurations and use them where
+      possible
+    + boards/stm32l0538-disco: add initial support
+    * stm32-common/spi: allow custom pin modes on spi to minimize power
+      consumption
+    * stm32l1/vendor: update vendor files to v2.3.0
+    * arduino/sketches: build sketches as a module
+    * pic32-wifire: add support for flashing with pic32prog
+
+CPU
+---
+    + cpu/esp32: lwIP netdev
+    * cpu/esp*: various improvements
+    * cpu/kinetis: enable HWRNG for k64f
+    - cpu/mips32r2_generic: remove cpu.
+    * cpu: Moved stdio_init() prior to periph_init() for ARM targets
+    * cpu/nrf5x: decelare radio_nrfble feat for all nrfs
```

Comment:
s/decelare/declare/ (probably present in original commit)

Reply:
Must have been from the original Italian. :-) Fixed in 652ec23, along with other misspellings.